src/features/auth/WalletAuthModal.tsx

```diff
@@ -101,6 +114,9 @@ export const WalletAuthModal = ({ open }: { open: boolean }) => {
     }
 
     try {
+      // after this succeeds, consumers of useWeb3React will re-run and
+      // web3Context.active will be true
+      setConnectMessage('');
```

Comment:
This actually doesn't work for us... since it hides the message for WalletConnect and Coinbase Wallet....